content/en/docs/combined-matrix.md

```diff
@@ -0,0 +1,342 @@
+| [GitLab](https://spinnaker.io/setup/artifacts/gitlab/)            | All supported versions    |                                                |
+| [Git Repo](https://spinnaker.io/setup/artifacts/gitrepo/)         | All supported versions    | Supports using the entire repo as an artifact. |
+| [Google Cloud Storage](https://spinnaker.io/setup/artifacts/gcs/) | All supported versions    |                                                |
+| [HTTP](https://spinnaker.io/setup/artifacts/http)                 | All supported versions    |                                                |
+| [Maven](https://spinnaker.io/setup/artifacts/maven/)              | All supported versions    |                                                |
+| [Oracle Object](https://spinnaker.io/setup/artifacts/oracle)      | All supported versions    |                                                |
+| [S3](https://spinnaker.io/setup/artifacts/s3/)                    | All supported versions    |                                                |
+
+## As code solutions
+
+### Pipelines as Code
+
+[![Generally available](/images/ga.svg)]({{< ref "release-definitions#ga" >}}) ![Armory](/images/armory.svg)
+
+[Pipelines as Code]({{< ref "install-dinghy" >}}) gives you the ability to manage your pipelines and their templates in source control.
+
+**Supported version control systems**
+
+| Feature   | Version | Armory version | Notes                  |
+|-----------|---------|---------------------------|------------------------|
+| BitBucket Cloud |         | All supported versions    |                        |
+| BitBucket Server | Previous two major versions |  All supported versions   |                        |
+| GitHub    |         | All supported versions    | Enterprise and GitHub.com |
+| GitLab    |         | All supported versions    | |
+
+**Features**
+
+| Feature                                                                           | Armory      | Notes |
+|-----------------------------------------------------------------------------------|------------------------|-------|
+| Modules                                                               | All supported versions | Templatize and re-use pipeline snippets across applications and teams |
+| Slack notifications                                                               | All supported versions |       |
+| Fiat service account integration                                                               | All supported versions |       |
+| Webhook secret validation                                                               | 2.20 or later |       |
+| Local modules for development                                                     | 2.20 or later          |       |
+| [Pull Request Validation]({{< ref "install-dinghy#pull-request-validations" >}}) | 2.21 or later          |       |
+
```

Comment:
We also support pub/sub
jenkins triggers
Jenkins CI
Fiat managed service account ingegration (replaces legacy "service account" stuff)

Reply:
@dogonthehorizon is the PUX team 👍  on officially supporting these?